core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1038,6 +1059,13 @@ class SparkContext(config: SparkConf) extends Logging {
    * operation will create many references to the same object.
    * If you plan to directly cache, sort, or aggregate Hadoop writable objects, you should first
    * copy them using a `map` function.
+   * @param path directory to the input data files, the path can be comma separated paths
+   * as a list of inputs
+   * @param minPartitions suggested minimum number of partitions for the resulting RDD
+   * @param km implicitly found Class of the key associated with the `fm` implicit parameter
```

Comment:
I ran `build/sbt unidoc` as described [here](https://github.com/apache/spark/blob/master/docs/README.md)
I got this output:
```
[info] 100 warnings
[info] Main Java API documentation successful.
```
Searching `/SparkContext.scala` through warnings didn't yield any results. So I guess it's fine.

This is how it looks in Scaladoc: [https://i.imgsafe.org/ea9b9e32bb.png](https://i.imgsafe.org/ea9b9e32bb.png)

Scala's Scaladocs [do contain](http://www.scala-lang.org/api/current/scala/collection/immutable/List.html#sorted[B>:A](implicitord:scala.math.Ordering[B]):Repr) implicit parameters descriptions, otherwise it would be hard to understand method's signature.

I hope this answers your questions, it's up to you to tell me if you want them removed or not.